src/services/index.ts

```diff
@@ -39,6 +40,8 @@ export const subscribeToTokenList = subscribeToTokenListFactory(apis)
 
 export const getTokens = getTokensFactory(apis, { getTokenFromErc20 })
 
+export const getTrades = getTradesFactory({ ...apis, getTokens })
```

Comment:
Why not use signature like in other services `(apis, { getTokens })`

Reply:
@anxolin commented in other PR, why use a different signature for this injection?

So I thought about adding everything into the same param. Refactor the others next.

Arguments in favor or against?

Reply:
Only argument -- consistency

Reply:
yes, for me it's a strange to add the dependencies like `getTradesFactory({ ...apis, getTokens })` instead of `getTradesFactory({ ..anyIocDepencency })`

But if it's how we did things, I'm ok in pro of consistency